autoapi/mappers/python/objects.py

```diff
@@ -1,6 +1,8 @@
 import collections
+import logging
```

Comment:
autoapi uses the sphinx logging module so that we get the log formatting provided by sphinx. You can look at some of the other modules to see how we use it.